Program.cs

```diff
@@ -1,7 +1,40 @@
 ﻿namespace Training {
-    internal class Program {
-        static void Main (string[] args) {
-            Console.WriteLine ("Hello, World!");
-        }
-    }
-}
\ No newline at end of file
+   internal class Program {
+      static void Main (string[] args) {
+         Console.Write ("Enter Password: ");
+         string Password = Console.ReadLine ();
+         if (check (Password) == "true") Console.WriteLine ("Strong password.");

```

Comment:
Can you check for the data type [bool](https://www.w3schools.com/cs/cs_booleans.php)?

Reply:
Changed.